Dockerfile

```diff
@@ -27,17 +31,25 @@ RUN apt-get update && \
         php5-mcrypt \
         php5-json \
         php5-xdebug \
-        nano && \
-    php5dismod xdebug
-
+        newrelic-php5 \
+        nano \
+        wget \
+        git \
+        && \
+    php5dismod xdebug && \
+    php5dismod newrelic
+
+# Build/install any extensions that aren't in trouble-free packaging
 RUN pecl install igbinary-1.2.1 && \
     echo 'extension=igbinary.so' > /etc/php5/mods-available/igbinary.ini && \
-    php5enmod igbinary
-
-RUN printf "\n" | pecl install apcu-4.0.7 && \
+    php5enmod igbinary && \
+    printf "\n" | pecl install apcu-4.0.7 && \
     echo 'extension=apcu.so' > /etc/php5/mods-available/apcu.ini && \
     php5enmod apcu
 
+# Prevent newrelic daemon from auto-spawning; uses newrelic run.d script to enable at runtime, when ENV variables are present
+# @see https://docs.newrelic.com/docs/agents/php-agent/advanced-installation/starting-php-daemon-advanced
+RUN sed -i "s/;newrelic.daemon.dont_launch = 0/newrelic.daemon.dont_launch = 3/" /etc/php5/mods-available/newrelic.ini
```

Comment:
newrelic starts automatically with php fpm in this mode (took me awhile to figure out)